types/react-leaflet/index.d.ts

```diff
@@ -1,235 +1,351 @@
-// Type definitions for react-leaflet 1.0
+// Type definitions for react-leaflet 1.1
 // Project: https://github.com/PaulLeCam/react-leaflet
-// Definitions by: Dave Leaver <https://github.com/danzel>
+// Definitions by: Dave Leaver <https://github.com/danzel>, David Schneider <https://github.com/davschne>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 2.2
 
 import * as Leaflet from 'leaflet';
 import * as React from 'react';
 
 // All events need to be lowercase so they don't collide with React.DOMAttributes<T>
 // which already declares things with some of the same names
 
-export interface LeafletLayerEvents {
-    onbaselayerchange?(event: Leaflet.LayersControlEvent): void;
-    onoverlayadd?(event: Leaflet.LayersControlEvent): void;
-    onoverlayremove?(event: Leaflet.LayersControlEvent): void;
-    onlayeradd?(event: Leaflet.LayerEvent): void;
-    onlayerremove?(event: Leaflet.LayerEvent): void;
-}
-export interface LeafletMapStateChangeEvents {
-    onzoomlevelschange?(event: Leaflet.Event): void;
-    onresize?(event: Leaflet.ResizeEvent): void;
+type Children = React.ReactNode | React.ReactNode[];
```

Comment:
Why did `Children` get introduced?
I see it gets used everywhere, but does it actually do anything? AFAIK what it is doing is default behaviour.

Would be nice if it could enforce proper children, like you can't put a `<div>` in a `<Map>`.